charts/internal/shoot-control-plane/templates/metallb.yaml

```diff
@@ -226,20 +388,19 @@ spec:
           allowPrivilegeEscalation: false
           capabilities:
             add:
-            - NET_ADMIN
             - NET_RAW
-            - SYS_ADMIN
             drop:
             - ALL
           readOnlyRootFilesystem: true
-      hostNetwork: false
+      hostNetwork: true
```

Comment:
Why did this change?

Reply:
Just copied over mostly the manifest from upstream, but good catch

Reply:
reset to false